frontend/src/app/components/wp-table/external-configuration/external-query-configuration.component.ts

```diff
@@ -1,35 +1,52 @@
-import {AfterViewInit, ChangeDetectorRef, Component, Inject, ViewChild} from '@angular/core';
+import {AfterViewInit, ChangeDetectorRef, Component, Inject, OnInit, ViewChild} from '@angular/core';
 import {WorkPackageEmbeddedTableComponent} from 'core-components/wp-table/embedded/wp-embedded-table.component';
 import {WpTableConfigurationService} from 'core-components/wp-table/configuration-modal/wp-table-configuration.service';
 import {RestrictedWpTableConfigurationService} from 'core-components/wp-table/external-configuration/restricted-wp-table-configuration.service';
 import {OpQueryConfigurationLocalsToken} from "core-components/wp-table/external-configuration/external-query-configuration.constants";
+import {UrlParamsHelperService} from "core-components/wp-query/url-params-helper";
 
 export interface QueryConfigurationLocals {
   service:any;
   currentQuery:any;
-  disabledTabs:{ [key:string]:string };
+  urlParams?:boolean;
+  disabledTabs?:{ [key:string]:string };
   callback:(newQuery:any) => void;
 }
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/9e0205465347a613a3b5b1319785373708e2af7f/frontend/src/app/components/wp-table/external-configuration/external-query-configuration.component.ts#L15"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>